planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/calculation.cpp

```diff
@@ -20,6 +20,13 @@
 
 namespace autoware::behavior_path_planner::utils::lane_change::calculation
 {
+
+rclcpp::Logger get_logger()
+{
+  constexpr const char * name{"lane_change.calculation"};
+  return rclcpp::get_logger(name);
```

Comment:
I think the logger can be made `static` (instead of creating a new temporary object each time).